pkg/constants/constants.go

```diff
@@ -85,6 +85,19 @@ const (
 	CallGitLabGroupGetVariable     = "GroupVariableGet"
 	CallGitLabGroupListVariables   = "GroupVariablesList"
 
+	ProviderAKEYLESSSM                  = "AKEYLESSLESS/SecretsManager"
+	CallAKEYLESSSMGetSecretValue        = "GetSecretValue"
+	CallAKEYLESSPSGetParametersByPath   = "GetParametersByPath"
+	CallAKEYLESSSMDescribeSecret        = "DescribeSecret"
```

Comment:
The REST API call is `DescribeItem`, i'd suggest to call it that as well to avoid confusion